src/components/common/HideTextInput/HideTextInput.js

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import { string, func } from "prop-types";
+import conditionalClassName from "classnames";
+
+import visibility from "assets/icons/visibility.png";
+import visibilityOff from "assets/icons/visibility-off.png";
+
+import "./hide-text-input.scss";
+
+const HideTextInput = ({ name, label, placeHolder, value, handleChange }) => {
+  const [showText, setShowText] = useState(false);
+
+  const visibilitySrc = showText ? visibility : visibilityOff;
+
+  return (
+    <label className="regular-text">
+      {label}
+      <div className="hide-text-input-container">
+        <input
+          name={name}
+          type={conditionalClassName({
+            text: showText,
+            password: !showText,
+          })}
+          placeholder={placeHolder}
+          className="hide-text-input regular-text"
+          value={value}
+          onChange={handleChange}
+        />
+        <button
+          type="button"
+          className="visibility-button"
+          onClick={() => setShowText(!showText)}
+        >
+          <img
+            src={visibilitySrc}
+            alt=""
```

Comment:
This alt text should not be empty. It should have something descriptive to ensure accessible technologies know what this means.
[Here you can see example 4](https://www.w3.org/WAI/tutorials/images/functional/)
I think it should be in function of the state of the text, if it's hidden it shows a different icon than when it's shown thus it should have a text for each case, is that right?